packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -111,7 +111,11 @@ class CollapsibleMenuView extends React.Component {
           break;
         }
 
-        hiddenStartIndex = i;
+        if (this.props.children.length !== undefined && this.props.children[i].type === CollapsibleMenuViewDivider) {
```

Comment:
`this.props.children.length !== undefined` check isn't really needed.

Reply:
this throws error on accessing children by index `this.props.children[i]` in few scenario's like CollapsibleMenuViewSingleItem where this.props.children will have single object.

Reply:
Ah, gotcha.

Reply:
Thank you,  yes React.Children.count() suits the purpose of this condition. Added comments above if :  c7ee6e81d451dbafd42cfcf8062038073eae9319